qiskit/pulse/schedule.py

```diff
@@ -104,8 +101,16 @@ def channels(self) -> Tuple[Channel]:
         return tuple(self._timeslots.keys())
 
     @property
-    def _children(self) -> Tuple[Tuple[int, ScheduleComponent], ...]:
-        return self.__children
+    def children(self) -> Tuple[Tuple[int, ScheduleComponent], ...]:
```

Comment:
I'm not convinced about this change. I'll try to keep reading. Maybe we should change the name-- `children` makes implications about the internal data structures in use. that breaks abstraction

Reply:
I agree with you, see my comment above in `instruction.py` and we can decide on which is the lesser of two evils right now 😄 .